### ISSUE TITLE
Spot-On-2023.08.05.zip Log: no ui connection to kernel possible

### ISSUE THREAD

**Issue by bolzobrazil** [textbrowser/spot-on#20]
2023-09-29
A UI error (No functional TLS backend was found) occurred.

2023-09-29
spoton_crypt::defaultSslCiphers(): empty cipher list.

2023-09-29
A UI error (No functional TLS backend was found) occurred.

2023-09-29
A UI error (QSslSocket::connectToHostEncrypted: TLS initialization failed) occurred.

2023-09-29
spoton::slotKernelSocketError(): socket error (TLS initialization failed).

**Comment by textbrowser** [comment]
OpenSSL 3 and Qt 6 and Windows 10, 11 behave entirely different. As such, Spot-On may only provide the services which are provided by the underlying system. MacOS has similar behavior. Since Spot-On behaves well in FreeBSD and Linux, the problem is not Spot-On. I do not have an interest in MacOS and Windows nor the resources to move Spot-On into their paradigms.

TL;DR

Set the kernel's key size to 0. You'll also encounter problems with listeners. Again, Windows is not the Windows it used to be.
